lib/commons/aria/get-element-unallowed-roles.js

```diff
@@ -1,5 +1,15 @@
 /* global aria */
 
+const dpubRoles = [
+	'doc-backlink',
+	'doc-biblioentry',
+	'doc-biblioref',
+	'doc-cover',
+	'doc-endnote',
+	'doc-glossref',
+	'doc-noteref'
+];
```

Comment:
Just to clarify: these are not all the DPUB ARIA roles, but the ones which are subclassing roles that are implicit on some native HTML elements (specifically: `img`, `link`, and `listitem`).

Maybe this can be clarified in a comment for later reference?